source/CardGames.py

```diff
@@ -155,4 +170,16 @@ def dealCards(self, numCards: int, players: "list[Player]"):
 
   def resetDeck(self):
     self.deck.reset()
-    self.deck.shuffle()
```

Comment:
what did you change here

Reply:
I was adding a line that would call the shuffle feature from above on to the deck feature.

Reply:
Where is four of a kind?

Reply:
it is at the bottom of my committed code, under the Game class.